source/_integrations/cloud.markdown

```diff
@@ -26,3 +26,46 @@ Once activated, go to the configuration panel in Home Assistant and create an ac
 ```yaml
 config:
 ```
+### Google Assistant configuration
```

Comment:
This part of the documentation is hosted at NabuCasa.com, maybe point to that location instead?

Reply:
Aha, I see! Good idea, I'll update with a link NabuCasa